van-gogh-second-page/jons/tickets.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tickets</title>
+    <link rel="stylesheet" href="./index-tickets.css" />
+    <!-- Download fonts if you prefer doing it from html side -->
+    <!-- <link rel="preconnect" href="https://fonts.gstatic.com"> -->
+    <!-- <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@500&display=swap" rel="stylesheet"> -->
+</head>
+
+<body>
+    <header>
+        <nav>
+            <a href="#" class="nav-tab">
+                Home
+            </a>
+            <a href="#" class="nav-tab">
+                Tickets
+            </a>
+            <a href="#" class="nav-tab">
+                Art and stories
+            </a>
+            <a href="#" class="nav-tab">
+                About
+            </a>
+            <a href="#" class="nav-tab">
+                Webstore
+            </a>
+            <a href="#" class="nav-tab">
+                NL | EN
+            </a>
+            <a href="#" class="nav-tab">
+                <!-- using bootstap icon to prevent fontawesome import -->
+                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-search" viewBox="0 0 16 16">
+                    <path d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"/>
+                </svg>
```

Comment:
I've just added it in order to remove fontawesome

Reply:
Great.